doc/guides/bluetooth/bluetooth-dev.rst

```diff
@@ -42,6 +42,33 @@ To start developing using this setup follow the :ref:`Getting Started Guide
 boards that support Bluetooth and then :ref:`run the application
 <application_run_board>`).
 
+.. _bluetooth-hci-tracing:
+
+Embedded HCI tracing
+--------------------
+
+When running both Host and Controller in actual Integrated Circuits, you will
+only see normal log messages on the console by default, without any way of
+accessing the HCI traffic between the Host and the Controller.  However, there
+is a special Bluetooth logging mode that converts the console to use a binary
+protocol that interleaves both normal log messages as well as the HCI traffic.
+Set the following Kconfig options to enable this protocol before building your
+application:
+
+.. code-block:: console
+
+   CONFIG_BT_DEBUG_MONITOR=y
+   CONFIG_UART_CONSOLE=n
+
+The first item replaces the :opt:`CONFIG_BT_DEBUG_LOG` option, and the second
```

Comment:
I think this one needs to be `:option:` (that's what CI complains about)

Reply:
Fixed, thanks!